rag/tests/test_rag.py

```
@@ -96,6 +96,23 @@ def test_retries_on_failure_then_succeeds(self):
         assert result == [[0.5]]
         assert mock_emb.embed_documents.call_count == 2
 
+    def test_retry_wait_is_capped(self):
+        """#994: 指数バックオフの待機時間が上限(5秒)で頭打ちになること
+        (スレッドプール専有時間を抑えるため)。上限なしなら2**5=32秒になるはずの
+        6回目の待機が5秒に切り詰められることを確認する。"""
+        p, mock_emb = self._patch_embeddings(
```

Ruff (0.16.1)

[warning] Unpacked variable `mock_emb` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)